core/templates/components/summary-tile/learner-topic-summary-tile.component.html

```diff
@@ -3,7 +3,7 @@
      target="{{ openInNewWindow ? '_blank' : '_self' }}"
      class="oppia-topic-link">
     <div class="oppia-title-section" [ngStyle]="{'background-color': thumbnailBgColor}">
-      <img class="oppia-thumbnail-image" [src]="thumbnailUrl">
+      <img alt="" class="oppia-thumbnail-image" [src]="thumbnailUrl">
```

Comment:
I will suggest to add a comment where ever `alt=""`  is being used. The comment should outline the importance of empty alt text being added. This will ensure that developers in future just don't simply remove it thinking that it serve no purpose.